public/index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Cool stuff" />
-    <title>WikiForest</title>
+    <title>Entitree</title>
```

Comment:
How about writing the name in the config?

Reply:
yes, the title will need to be updated anyway from js like document.title = "itemLabel - propLabel - siteName". will open an issue for that